jade-repl.el

```diff
@@ -220,6 +221,82 @@ LEVEL is a string representing the logging level, it can be
         ;; convenience
         (when error (message string))))))
 
+(defun jade-repl-emit-console-message-with-params (string params level url line)
+  "Emit a console message STRING.
```

Comment:
`jade-repl-emit-console-message-with-params`'s parameters are growing.  What about unifying this function with `jade-repl-emit-console-message`, and pass a map `message' instead of parameters?